pkgs/by-name/pr/protonmail-desktop/package.nix

```diff
@@ -0,0 +1,95 @@
+{ lib, stdenv, fetchurl, wrapGAppsHook, makeWrapper, autoPatchelfHook, pkgs
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, dpkg
+, glib
+, gtk3
+, libX11
+, libXtst
+, libdrm
+, mesa
+, nss
+, udev
+, xdg-utils
+, coreutils
+, xorg
+
+# command line arguments which are always set e.g "--disable-gpu"
+, commandLineArgs ? ""
+
+# Necessary for USB audio devices.
+, pulseSupport ? stdenv.isLinux
+, libpulseaudio
+
+# For GPU acceleration support on Wayland (without the lib it doesn't seem to work)
+, libGL
+
+# For video acceleration via VA-API (--enable-features=VaapiVideoDecoder,VaapiVideoEncoder)
+, libvaSupport ? stdenv.isLinux
+, libva
+}:
+
+let
+  inherit (lib) optional makeLibraryPath makeSearchPathOutput makeBinPath;
+
+  deps = [
+    alsa-lib at-spi2-atk at-spi2-core atk
+    glib gtk3 libdrm libX11 libGL
+    libXtst mesa nss udev xorg.libxcb
+  ]
+    ++ optional pulseSupport libpulseaudio
+    ++ optional libvaSupport libva;
+
+  rpath = makeLibraryPath deps + ":" + makeSearchPathOutput "lib" "lib64" deps;
+  binpath = makeBinPath deps;
+in
+stdenv.mkDerivation rec {
+  pname = "ProtonMail-desktop";
```

Comment:
```suggestion
  pname = "protonmail-desktop";
```

Reply:
changed to proton-mail so it's usable in the URL and becaus that's the main program delivered. No conflicts with other packages. Maintainer change extracted to separate commit.